### PR TITLE
Kuszcjan/lesson 04 unit testing

### PR DESCRIPTION
**Pull request by mielcszy**

### CODE REVIEW

**Comment by kuszcjan** [comment]
pls do not forgot to @mention

**Comment by mielcszy** [comment]
@kuszcjan: done - please review

In task 2, you did not specify whether it should be unittest or pytest so I used unittest but in the 3rd task pytest instead ;)

File: students/mielczarek_szymon/lesson_04_unit_testing/test_distance.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        self.assertEqual(distance(-2, -7, -13, -1), 12.529964086141668)
+
+    def test_vertical_distance(self):
+        self.assertEqual(distance(2, 2, 2, 12), 10)
+
+    def test_horizontal_distance(self):
+        self.assertEqual(distance(2, 2, 12, 2), 10)
+
+    def test_typical_coordinates(self):
+        self.assertEqual(distance(2, 3, 12, 4), 10.04987562112089)
```
Comment (kuszcjan) [inline students/mielczarek_szymon/lesson_04_unit_testing/test_distance.py]: I know it wasn't specified, but there is no test scenario with float numbers as input

File: students/mielczarek_szymon/lesson_04_unit_testing/test_collatz_sequence.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import sys
+import unittest
+from lesson_03_functions.the_collatz_sequence import collatz
+sys.path.append('..')
```
[the remark this answers is not in the document]
Comment (mielcszy) [reply students/mielczarek_szymon/lesson_04_unit_testing/test_collatz_sequence.py]: I had so at the very beginning but the PyCharm was complaing that: "PEP 8: module level import not at top of file". I changed it, as it is now, and it worked, at least in PyCharm env :) But indeed when I run from bash terminal I got ModuleNotFoundError.
Will fix it.

File: students/mielczarek_szymon/lesson_04_unit_testing/test_distance.py (hunk 1 above)
Comment (mielcszy) [reply students/mielczarek_szymon/lesson_04_unit_testing/test_distance.py]: But there is no float inputs distance(2, 3, 12, 4), only outputs :)

File: students/mielczarek_szymon/lesson_04_unit_testing/test_collatz_sequence.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (kuszcjan) [reply students/mielczarek_szymon/lesson_04_unit_testing/test_collatz_sequence.py]: Sometimes it's better to ignore PyCharm complains. Sometimes it's necessary :)

**Pull request merged**